cmd/thanos/query.go

```diff
@@ -129,6 +134,9 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 			*caCert,
 			*serverName,
 			*httpBindAddr,
+			*webRoutePrefix,
```

Comment:
Worth to move this madness to struct at some point ;p

Reply:
A new "TODO" is added in func runQuery().